static/app/components/profiling/boundTooltip.tsx

```diff
@@ -109,7 +110,9 @@ function BoundTooltip({
         fontFamily: flamegraphTheme.FONTS.FONT,
         left: cursorHorizontalPosition,
         top: cursorVerticalPosition,
-        width: Math.min(tooltipRect.width, bounds.width - cursorHorizontalPosition - 2),
+        width:
+          Math.min(tooltipRect.width, bounds.width - cursorHorizontalPosition - 2) +
+          PADDING,
```

Comment:
What's this extra padding for?

Reply:
The text measure only measures the internal text of the tooltip, but because I added the padding to our tooltip because it had none before and because we use box-sizing: border-box, the "content box" width is actually smaller by 2x the horizontal padding. I think I want to actually refactor this so we dont rely on the content, but on the actual rendered width and then we no longer need this + we can render whatever we want inside the tooltip